### PR TITLE
fix "Regulatory Attachments" button in the top bar

### PR DESCRIPTION
**Pull request by usrtim**

### CODE REVIEW

**Review by elpoelma — CHANGES_REQUESTED** [review]
The `AuMainContainer` component should be wrapping the content of the page (so the `{{outlet}}`) and not the header.

File: app/templates/application.hbs [hunk 1]
```diff
@@ -1,28 +1,32 @@
-  {{page-title  
-    (concat
-        (t "utility.app-name") (if this.showEnvironment (concat ' - ' this.environmentName))
-      )
-  }}
+{{page-title
+  (concat
+    (t "utility.app-name") (if this.showEnvironment (concat ' - ' this.environmentName))
+  )
+}}
```
[the remark this answers is not in the document]
Comment (abeforgit) [reply app/templates/application.hbs:5]: an autoformatter most likely
We use prettier in our projects, but for .hbs files I'm not 100% sure if prettier handles those
We're not particularly strict about it as long as things are readable, but we do recommend formatting with prettier

[the remark this answers is not in the document]
Comment (usrtim) [reply app/templates/application.hbs:5]: @abeforgit yes, I'm also using prettier but I'm not sure what happend there

**Review by elpoelma — APPROVED** [review]

File: app/templates/application.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (abeforgit) [reply app/templates/application.hbs:5]: well its just that this file is old and was never run through prettier

[the remark this answers is not in the document]
Comment (usrtim) [reply app/templates/application.hbs:5]: yes, might be

**Pull request merged**